src/status_im2/contexts/chat/home/view.cljs

```diff
@@ -22,11 +22,12 @@
   [_ index]
   #js {:length 56 :offset (* 56 index) :index index})
 
-(defn filter-items-by-tab
+(defn filter-and-sort-items-by-tab
   [tab items]
-  (if (= tab :tab/groups)
-    (filter :group-chat items)
-    (filter :chat-id items)))
+  (let [key (if (= tab :tab/groups) :group-chat :chat-id)]
+    (->> items
+         (filter key)
+         (sort-by :timestamp >))))
```

Comment:
Are we sure all possible chat items have a `:timestamp` key that is not `nil`?

Reply:
If there is a possibility of having `nil` timestamps, I believe it would be appropriate to display those items last. In such cases, should we consider adding additional sorting parameters? If not necessary, I believe the current implementation should suffice.

Reply:
> Are we sure all possible chat items have a `:timestamp` key that is not nil?

@ibrkhalil - As far as I checked, there is no possibility of `nil` for `:timestamp`, even for newly accepted contact requests or joined groups.

Reply:
Nice, Thanks Mohamed!